### DISCUSSION TITLE
I am a remote, online student and my application is denied multiple times.

### DISCUSSION THREAD

**Discussion by AmaKbe** [community/community#65047] [GitHub Education]
### Hi everyone,

Hi, 

I am a remote, online student, and I am applying to the global campus program for a class. I am applied multiple times, and my location seems to be a problem.  The one document I have that doesn't get the other error message for my application, like legal name etc..., is my syllabus. But, when I upload my syllabus, it says my location does not match my campus's location. However, in the same error message, it says that if it is an online/virtual education, it should mention it on the document. But, my syllabus clearly indicates its an online class. I don't know what the problem is, so I opened a support ticket. Since then, I have gotten a single response for two days. This is my last resort, since I am doing this for a class. Can someone please help me?

**Comment by arafajs** [comment]
Try uploading a different document that clearly states that you are an online student. This could be a letter from your school, a transcript, or any other document that shows that you are taking classes online.

**Reply by AmaKbe** [reply]
I have entered multiple documents that indicate I am an online students. Some of the words include "online" and "virtual". I even included an email that clearly state I am an online student directly. However, my application is constantly being declined.

**Comment by lab596** [comment]
1. Contact Your Instructor:
Reach out to your class instructor or professor and explain the situation to them. They might be able to provide additional information or support that can help clarify the online nature of the class. They could potentially provide a letter or statement confirming the online format of the class.

2. Follow Up on Support Ticket:
While it's frustrating that you haven't received a response for your support ticket, it's important to give the support team some time to address your issue. You can consider sending a polite follow-up message to inquire about the status of your ticket. Remember that support teams can be busy, so a little patience can go a long way.

3. Seek Alternative Contact Channels:
If the support ticket doesn't yield a timely response, check if there are alternative contact channels available, such as a phone number or a live chat option. Some platforms offer multiple ways to reach their support teams, and trying different avenues might help you get a faster response.

**Reply by AmaKbe** [reply]
It worked! Thank you very much. 
Ps: Sorry for the late reply! I have been very busy.

**Reply by lab596** [reply]
Thats great to hear, if you get a chance can you mark this answer as correct so others who have a similar problem can refer back to this discussion. Thanks!

**Comment by AmaKbe** [comment]
Update: I have now submitted a successful application. Thank you for everyone who helpled me. I really appreciate it.
